### DISCUSSION TITLE
The subcategories should appear on the parent category page,

### DISCUSSION THREAD

**Discussion by KBM90** [bagisto/bagisto#9391] [Ideas]
The subcategories should appear on the parent category page, as it was in the first version. Now, I can't see this feature, as all categories appear individually. For example, Category A, which has several subcategories, when clicked, displays a page indicating no products belong to this category, while it should show the subcategories belonging to it. Clicking on a subcategory should then display the list of products.

**Comment by amit-webkul** [comment]
Hi @KBM90,

Thanks for bringing this up, and we really appreciate your concern and efforts toward Bagisto.

We're planning to add this feature in a new way in our future releases. Just stay tuned for updates!

Thanks again.

**Comment by hrabosh** [comment]
Hello @amit-webkul , I would like to ask if the date (at least approximate) or the version in which this function could be added is known. I'm totally waiting for her. Thank you and have a nice day :)
